hudi-client/hudi-client-common/src/main/java/org/apache/hudi/io/HoodieMergeHandleFactory.java

```diff
@@ -79,6 +83,7 @@ public static <T, I, K, O> HoodieMergeHandle<T, I, K, O> create(
       HoodieBaseFile dataFileToBeMerged,
       TaskContextSupplier taskContextSupplier,
       Option<BaseKeyGenerator> keyGeneratorOpt) {
+    LOG.info("Get updateHandle for fileId " + fileId + " and partitionPath " + partitionPath + " at commit " + instantTime);
```

Comment:
tip: with slf4j, you can aso log as
`LOG.info("Get updateHandle for fileId {} and partitionPath {} at commit {}", fileId, partitionPath, instantTime);`